python/ray/_private/event/export_event_logger.py

```diff
@@ -12,24 +13,47 @@
 from ray.core.generated.export_submission_job_event_pb2 import (
     ExportSubmissionJobEventData,
 )
+from ray.core.generated.export_train_state_pb2 import (
+    ExportTrainRunEventData,
+    ExportTrainRunAttemptEventData,
+)
 from ray._private.protobuf_compat import message_to_dict
 
 global_logger = logging.getLogger(__name__)
 
 # This contains the union of export event data types which emit events
 # using the python ExportEventLoggerAdapter
-ExportEventDataType = Union[ExportSubmissionJobEventData]
+ExportEventDataType = Union[
+    ExportSubmissionJobEventData,
+    ExportTrainRunEventData,
+    ExportTrainRunAttemptEventData,
+]
+
+
+class EventLogType(Enum):
```

Comment:
Can you add a comment mentioning this represents the valid file names?

Reply:
Added a docstring to the init